### PR TITLE
Add some missing steps, update yarn build

### PR DESCRIPTION
**Pull request by csm10495**
Add some steps that you need before running yarn. Also update yarn build to get further with Windows and Linux

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "A native desktop implementation of Pulse",
   "main": "built/main.js",
   "scripts": {
-    "build": "tsc && cp -r src/assets built/assets",
+    "build": "tsc && (cp -r src/assets built/assets || copy /b src/asserts built/assets)",
```
Comment (csm10495) [inline package.json]: With ref to https://github.com/klinker-apps/pulse-sms-issues/issues/949

**Comment by csm10495** [comment]
Hmm, even after this i hit this weird error (in yarn start): 

image

Seems like tsc is adding a character to the end of decrypt.js... any idea on that?

**Comment by klinker24** [comment]
Sorry, I am not really sure. I just tried it on my Windows computer, cleaned out my node modules and everything, I did not see any build issues. `yarn start` worked correctly on my end. Seems like something with your typescript if it isn't working. You do not have to install typescript globally, it is included with this project.

**Comment by csm10495** [comment]
@klinker24 , got it. Latest worked for me completely. Something about the copy args being off (but me not noticing), seems to be what was leading to the character being added. Please check what we have now.

File: package.json [hunk 2]
```diff
@@ -77,7 +77,7 @@
     "electron-builder": "22.1.0",
     "electron-notarize": "^0.1.1",
     "tslint": "^5.18.0",
-    "typescript": "^3.7.2"
+    "typescript": "3.7.4"
```
Comment (csm10495) [inline package.json]: I think this can be reverted back. Was from debug

**Review by klinker24 — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -17,10 +17,10 @@ The desktop app is built on top of Electron's `BrowserView`, rather than Chromiu
 
 ## Build Instructions
 
-1. Install Electron and project dependencies:
+1. Install Electron, Yarn, Typescript, and project dependencies:
```
Comment (klinker24) [inline README.md]: Actually, let's update this comment.

```suggestion
1. Use Yarn as the package manager:
```

**Review by klinker24 — CHANGES_REQUESTED** [review]

**Comment by csm10495** [comment]
I think I've made the requested change.

**Review by klinker24 — APPROVED** [review]

**Pull request merged**